### PR TITLE
Fall back to pytesseract if paddleocr fails

### PR DESCRIPTION
**Pull request by rbiseck3**

### CODE REVIEW

**Review by cragwolfe — COMMENTED** [review]
The changes make sense to me!

**Comment by benjats07** [comment]
LGTM

**Review by benjats07 — APPROVED** [review]

File: unstructured_inference/__version__.py
Comment (benjats07) [inline unstructured_inference/__version__.py:1]: Probably we want 0.5.3-dev, but not sure if we are going to release this version.
What do you think @cragwolfe?

**Review by benjats07 — APPROVED** [review]

File: unstructured_inference/__version__.py
Comment (qued) [reply unstructured_inference/__version__.py:1]: Yeah this should either be 0.5.2 (if we are ready to release) or 0.5.2-dev2.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## 0.5.3
+## 0.5.2-dev2
 
 * Configure fallback for OCR token if paddleocr doesn't work to use tesseract
```
Comment (qued) [inline CHANGELOG.md]: The pattern is to keep all changes consolidated until we do a release. So this section should be removed, the change should be listed in the `0.5.2-dev1` section, and then the `0.5.2-dev1` header should be bumped to `0.5.2-dev2`.

File: unstructured_inference/__version__.py
Comment (cragwolfe) [reply unstructured_inference/__version__.py:1]: i think we are ready to release! so that we can bring it into unstructured and then unstructured-api

**Pull request merged**